ckan/plugins/interfaces.py

```diff
@@ -39,6 +39,7 @@
     u'IBlueprint',
     u'IPermissionLabels',
     u'IForkObserver',
+    u'ICLICommands',
```

Comment:
It's my personal opinion - such sequences of capitalized letters are just ugly. There are some styles that suggest using title-case for capitalized sequences that consist of more than 3 letters, i/e `ICliCommand`. What do you think?

Reply:
I can't find the reference but I think ICLICommand is correct by python naming standards in the same way as HTMLParser is preferred over HtmlParser

Reply:
What about ICommands or IClick then?

Reply:
:+1: